static/app/utils/events.tsx

```diff
@@ -61,7 +61,9 @@ export function getTitle(
 
   if (type === 'error') {
     result.subtitle = culprit;
-    if (metadata.type) {
+    if (metadata.current_tree_label) {
+      result.title = metadata.current_tree_label.join(' | ');
```

Comment:
This crashed when I sent in a Rust sample event, because of this line (label was string not array):

https://github.com/getsentry/sentry/pull/26740/files#diff-d9cbc1735687707e9f44acb1e7af9adb3325bf9d666854cb513df49ccae6836dL8

Reply:
Thanks, fixed